### DISCUSSION TITLE
Semi-Automatic Classification Plugin - QGIS

### DISCUSSION THREAD

**Discussion by rgdepaula** [semiautomaticgit/SemiAutomaticClassificationPlugin#200] [Q&A] [answered]
Hi gys,

I'm with a problem with Semi-Automatic Classification QGIS Plugin in my mac as you can see the message error attached. My mac is an intel processor and it is with BigSur installed. My QGIS is the LTR version 3.16.13-Hannover. I'm trying run the plugin with Landsat 8 imagens, but this error is appearing always i turn on the QGIS and it doesn't run the plugin. Stop in 20% calculating. Could you please help me to correct this problem?

Thanks.

<img width="586" alt="Captura de Tela 2021-11-13 às 15 54 15" src="https://user-images.githubusercontent.com/87235938/141655839-a76171f9-22e7-42fe-ae5b-f8e1960ad26e.png">

<img width="1280" alt="Captura de Tela 2021-11-13 às 15 41 33" src="https://user-images.githubusercontent.com/87235938/141655843-36827332-510d-41c8-9cf7-f4e5b99f2af0.png">

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @rgdepaula please follow this guide to install QGIS https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#i-have-installation-issues-on-mac-os-how-can-i-install-scp-on-mac-os
